tools/SourceKit/tools/sourcekitd-test/sourcekitd-test.cpp

```diff
@@ -406,10 +407,24 @@ static int handleJsonRequestPath(StringRef QueryPath) {
     free(Err);
     return 1;
   }
-  sourcekitd_request_description_dump(Req);
+  if (Print) {
+    sourcekitd_request_description_dump(Req);
+  }
+  static size_t ReqCount = 0;
+  using namespace std::chrono;
+  auto t1 = high_resolution_clock::now();
   sourcekitd_response_t Resp = sourcekitd_send_request_sync(Req);
+  auto t2 = high_resolution_clock::now();
+  auto Span = duration_cast<nanoseconds>(t2 - t1);
+  if (ReqCount++ == 0) {
+    llvm::outs() << Span.count() << "\n";
+  } else {
+    llvm::outs() << "    " << Span.count() << "\n";
+  }
```

Comment:
Some thoughts:
* We should also wrap the regular (non-json-request-path) code-path with timing.
* The output format is idiosyncratic.  Can we put a marker at the beginning/end instead of indenting the 2nd+ request?  I'm assuming this is so if you make multiple invocations you can distinguish the invocations from the individual requests.
* Could we print in milliseconds or *maybe* microseconds?  Nanoseconds is overkill for all of our requests.
* Let's put the time unit in the output.